go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/giantswarm/apiextensions/v2 v2.5.1
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/certs/v3 v3.0.0
+	github.com/giantswarm/certs/v3 v3.0.1-0.20200925114212-d08b0085d6b0
```

Comment:
Let's tag that before merge. ❤️

Reply:
Sure thing, I'll update as soon as I release merged certs.